.storybook/webpack.config.js

```diff
@@ -38,8 +38,9 @@ module.exports = async ({ config, mode }) => {
     test: /\.s[ac]ss$/i,
     use: [
       'style-loader',
-      'css-loader',
-      'sass-loader'
+      { loader: 'css-loader', options: { importLoaders: 1 } },
+      'postcss-loader',
```

Comment:
adding postcss-loader because we do use inline svgs in the styles for video player icons.  in the documentation, these weren't showing